README.md

```diff
@@ -39,13 +39,13 @@ For details on usage, please see [the yumrepo puppet docs](https://puppet.com/do
 
 ## Reference
 
-Please see REFERENCE.md for the reference documentation.
+Please see [REFERENCE.md](https://github.com/puppetlabs/puppetlabs-yumrepo_core/blob/master/REFERENCE.md) for the reference documentation.
```

Comment:
The other module PRs didn't have a link to github, so I'm thinking this one shouldn't either